src/main/res/values/strings.xml

```diff
@@ -749,33 +749,33 @@
 
     <!-- 2016-04-03 -->
     <string name="error_tor_not_installed">Tor will not work unless Orbot is installed. Install now?</string>
-    <string name="error_tor_setting_failed">Failed to set proxy.</string>
+    <string name="error_tor_setting_failed">Could not set proxy.</string>
 
     <!-- 2016-04-10 -->
     <string name="error_parse_imgur_title">Parse Error</string>
-    <string name="error_parse_imgur_message">Imgur sent some data I couldn\'t understand.</string>
+    <string name="error_parse_imgur_message">Imgur sent incomprehensible data.</string>
 
     <string name="error_upload_fail_imgur_title">Upload failed</string>
-    <string name="error_upload_fail_imgur_message">The upload was rejected by Imgur for an unknown reason.</string>
+    <string name="error_upload_fail_imgur_message">Rejected by Imgur for unknown reason.</string>
 
     <string name="upload_to_imgur">Upload to Imgur</string>
     <string name="no_file_selected">No file selected</string>
     <string name="button_upload">Upload</string>
     <string name="button_browse">Browse…</string>
 
     <string name="error_file_too_big_title">File too big</string>
-    <string name="error_file_too_big_message">The file size (%1$s) is larger than the upload limit (%2$s).</string>
+    <string name="error_file_too_big_message">The %1$s file size has to be smaller than the upload limit of %2$s.</string>
 
-    <string name="error_file_open_failed_title">Failed to open file</string>
+    <string name="error_file_open_failed_title">Could not open file</string>
     <string name="error_file_open_failed_message">An error occurred while trying to open the file.</string>
 
     <string name="image_selected_summary">Image selected: %1$dx%2$d, %3$s</string>
 
 	<!-- 2016-06-10 -->
 	<string name="submit_post_specify_subreddit">Specify subreddit</string>
-	<string name="submit_post_title_empty">Title cannot be empty</string>
-	<string name="submit_post_url_empty">URL cannot be empty</string>
-	<string name="submit_post_self_text_hint">Self Text</string>
+	<string name="submit_post_title_empty">Add a title first</string>
+	<string name="submit_post_url_empty">Add a URL first</string>
+	<string name="submit_post_self_text_hint">Self-post text</string>
```

Comment:
This is the type of garbage English has instead of proper compound word rules. I dropped the optional hyphen in web-browser, but this will almost certainly fail in translation.